portal-impl/src/com/liferay/portal/model/impl/UserImpl.java

```diff
@@ -996,7 +1005,9 @@ protected String getProfileFriendlyURL() {
 
 	private boolean _isRequirePasswordReset() {
 		if (!isPasswordReset() ||
-			((_passwordPolicy != null) && !_passwordPolicy.isChangeable())) {
+			((_passwordPolicy != null) &&
+			 (!_passwordPolicy.isChangeable() ||
+			  !_passwordPolicy.isChangeRequired()))) {
```

Comment:
I think this password policy property only makes sense in the context of the first login since the password was last changed. It is to force a change of password after an admin has set one on the user's behalf. See the UI and other usages.

Reply:
Ah, yup you are correct again.  I've never considered the need for an admin to force a user to reset their password, regardless of what the default password policy is.  I'll remove this logic.